src/v2/kubernetes_cluster/proof/controller_runtime_safety.rs

```diff
@@ -91,6 +91,7 @@ pub open spec fn state_comes_with_a_pending_request(self, controller_id: int, st
 }
 
 // TODO: Investigate flaky proof.
+#[verifier(rlimit(4000))]
```

Comment:
At some point we should seriously look into these proofs that require 4000(!) rlimit

Reply:
I doubt anything actually requires 4000 rlimit, perhaps a kind of binary search could lower it. In terms of practical user experience, it's honestly not bad.